Source/Extensions/Blazorise.DataGrid/_DataGridRowCommand.razor

```diff
@@ -103,3 +104,11 @@ else if ( EditState == DataGridEditState.None && ParentDataGrid.IsCommandVisible
         </Div>
     </TableRowCell>
 }
+else if ( EditState == DataGridEditState.None && !ParentDataGrid.IsCommandVisible && ( ( ParentDataGrid.EditMode == DataGridEditMode.Inline || ParentDataGrid.EditMode == DataGridEditMode.Cell ) ) )
+{
+
+        <TableRowCell @onclick:stopPropagation="@(ParentDataGrid.CommandColumn?.PreventRowClick ?? false)" Class="@(Column.CellClass?.Invoke(Item))" Style="@BuildCellStyle()" TextAlignment="@Column.TextAlignment" VerticalAlignment="@Column.VerticalAlignment" Display="@Column.Display" Flex="@Column.Flex" Gap="@Column.Gap">
+            <Div Flex="@DefaultFlex" Gap="@DefaultGap">

```

Comment:
Why is there an empty Div here?

Reply:
Use case where:
Inline or Cell mode, with **New Command** Disabled, but Save/Cancel commands are enabled. This means that an empty command column needs to be rendered in order for the columns to align properly. So basically an empty column.

Reply:
Actually when writing this... I figured this is even a problem in the current version with just **Inline** mode,

https://blazorise.com/docs/extensions/datagrid/features/editing
Change to **Inline** and click edit, and notice the column alignment.

Reply:
Understand. In that case, place an `Span` instead. It is an `inline block` by default so it makes more sense in this context.